extensions/notebook/src/dialog/managePackages/addNewPackageTab.ts

```diff
@@ -0,0 +1,264 @@
+	private readonly SearchButtonLabel = localize('managePackages.searchButtonLabel', "Search");
+	private readonly PackageNameTitle = localize('managePackages.packageNameTitle', "Package Name");
+	private readonly PackageVersionTitle = localize('managePackages.packageVersionTitle', "Package Version");
+	private readonly PackageSummaryTitle = localize('managePackages.packageSummaryTitle', "Package Summary");
+
+	private readonly PackageNotFoundError = localize('managePackages.packageNotFound', "Could not find the specified package.");
+	private readonly NoVersionsFoundError = localize('managePackages.noVersionsFound', "Could not find any valid versions for the specified package.");
+
+	constructor(private dialog: ManagePackagesDialog, private jupyterInstallation: JupyterServerInstallation) {
+		this.addNewPkgTab = azdata.window.createTab(this.AddNewTabTitle);
+
+		this.addNewPkgTab.registerContent(async view => {
+			this.newPackagesSearchBar = view.modelBuilder.inputBox()
+				.withProperties<azdata.InputBoxProperties>({
+					placeHolder: this.SearchBarPlaceholder
+				}).component();
+
+			this.packagesSearchButton = view.modelBuilder.button()
+				.withProperties<azdata.ButtonProperties>({
+					label: this.SearchButtonLabel,
+					width: '80px'
+				}).component();
+			this.packagesSearchButton.onDidClick(() => {
+				this.loadNewPackageInfo();
+			});
+
+			this.newPackagesName = view.modelBuilder.text().withProperties({
+				value: this.InvalidTextPlaceholder
+			}).component();
+			this.newPackagesNameLoader = view.modelBuilder.loadingComponent()
+				.withItem(this.newPackagesName)
+				.withProperties({ loading: false })
+				.component();
+
+			this.newPackagesVersions = view.modelBuilder.dropDown().withProperties({
+				value: this.InvalidTextPlaceholder,
+				values: [this.InvalidTextPlaceholder]
+			}).component();
+			this.newPackagesVersionsLoader = view.modelBuilder.loadingComponent()
+				.withItem(this.newPackagesVersions)
+				.withProperties({ loading: false })
+				.component();
+
+			this.newPackagesSummary = view.modelBuilder.text().withProperties({
+				value: this.InvalidTextPlaceholder
+			}).component();
+			this.newPackagesSummaryLoader = view.modelBuilder.loadingComponent()
+				.withItem(this.newPackagesSummary)
+				.withProperties({ loading: false })
+				.component();
+
+			this.packageInstallButton = view.modelBuilder.button().withProperties({
+				label: this.InstallButtonText,
+				enabled: false,
+				width: '80px'
+			}).component();
+			this.packageInstallButton.onDidClick(() => {
+				this.doPackageInstall();
+			});
+
+			let formModel = view.modelBuilder.formContainer()
+				.withFormItems([{
+					component: this.newPackagesSearchBar,
+					title: ''
+				}, {
+					component: this.packagesSearchButton,
+					title: ''
+				}, {
+					component: this.newPackagesNameLoader,
+					title: this.PackageNameTitle
```

Comment:
I think you can just inline the localize calls here, that's what we do elsewhere in the code to tighten the classes up.